third_party/amd/backend/compiler.py

```diff
@@ -121,17 +121,17 @@ def make_ttgir(mod, metadata, options):
         pm = ir.pass_manager(mod.context)
         pm.enable_debug()
         passes.ttgpuir.add_coalesce(pm)
-        amd.passes.ttgpuir.add_remove_layout_conversions(pm)
```

Comment:
Can we delete the pass then? It should be dead code now

Reply:
It's dead now. I'll remove it.